internal/command/ssh/issue.go

```diff
@@ -42,6 +41,11 @@ validity.`
 
```

Comment:
positional arguments are not valid now

Reply:
Both args and flags should be valid?

Reply:
I'd opt not to be backwards compatible with this command.

But I am referring to line 40, we have to change it `cobra.NoArgs`:
```
 	cmd.Args = cobra.MaximumNArgs(3)
```